### DISCUSSION TITLE
Python script to convert steamguard-cli maFiles to SDA maFiles format

### DISCUSSION THREAD

**Discussion by infirms** [dyc3/steamguard-cli#260] [Show and tell]
### Description

Hi, I made a simple python script to convert steamguard-cli maFile format to SDA maFile format. This can be useful if you originally set up 2FA via steamguard-cli, but you still need the SDA maFile format file. From my testing it worked perfectly even for accepting trades, but there are some drawbacks regarding `server_time` and whole `Session` part of the maFile, this data is unknown from data it's converting it this might somehow break some things, but I haven't encountered any problems yet while converting ~5-10 accounts for testing even in original SDA.

Here is the script: https://gist.github.com/infirms/a057b7d695ff2b5b71fe86dc53688dd2

Use it like: `python cli_to_mafile.py <cli-mafile> <output-mafile>`

**Comment by dyc3** [comment]
Your link 404s, I think you need to make the gist public

**Reply by infirms** [reply]
Oh, is it working now?

**Reply by dyc3** [reply]
Yup!
